File: RealEstateAnalysis.Web/ClientApp/src/app/modules/location-analysis/dtos/writes/writeNeighborhood.ts
export class WriteNeighborhood {
    constructor(public neighborhoodName: string,
        public city: string,
        public medianHouseholdIncome: number,
        public medianContractRent: number,
        public cityUnemploymentRate: number,
        public neighborhoodUnemploymentRate: number,
        public povertyRate: number,
        public ethnicMixLargestSlicePercent: number,
        public homesMedianDaysOnMarket: number,
        public stateId: number) { }
}

File: RealEstateAnalysis.Domain/DTOs/Writes/LocationAnalysis/WriteNeighborhood.cs
namespace RealEstateAnalysis.Domain.DTOs.Writes.LocationAnalysis
{
    public class WriteNeighborhood
    {
        public string NeighborhoodName { get; set; }

        public string City { get; set; }

        public decimal MedianHouseholdIncome { get; set; }

        public decimal MedianContractRent { get; set; }

        public decimal CityUnemploymentRate { get; set; }

        public decimal NeighborhoodUnemploymentRate { get; set; }

        public decimal PovertyRate { get; set; }

        public decimal EthnicMixLargestSlicePercent { get; set; }

        public int HomesMedianDaysOnMarket { get; set; }

        public long StateId { get; set; }
    }
}